drivers/storage/portworx/component/storage_class.go

```diff
@@ -0,0 +1,198 @@
+package component
```

Comment:
nit: can rename this to portworx_sc.go to make it similar to others
OR
you can completely get rid of the portworx_ prefix as the pkg hierarchy already has portworx in it